packages/sync/src/config/SyncConfig.ts

```diff
@@ -95,4 +101,23 @@ export class SyncConfig implements DataSyncConfig {
     return Object.assign(this, clientOptions);
   }
 
+  /**
+   * A method to setup the storage. Defaults to IndexedDB wrapped by LocalForage if none is passed
+   * @param clientOptions the options passed from the constructor containing the storage options
+   */
+  private setStorage(clientOptions?: DataSyncConfig) {
+    LocalForage.config(this.storageOptions);
```

Comment:
This is not needed.

Reply:
You have to call config before you call createInstance or else you risk getItem, setItem etc not working correctly.